### DISCUSSION TITLE
How to install recognize correctly?

### DISCUSSION THREAD

**Discussion by DenuxPlays** [nextcloud/recognize#790] [Q&A] [answered]
Hello,
I installed recognized on my nextcloud aio instance, but I keep getting these errors:
![image](https://user-images.githubusercontent.com/83671398/230396798-6ee9d722-e556-423b-98b7-dcf3534aabfd.png)
I can not figure out why and how I can resolve this issue.
Can somebody help me please?

Note: Do you need some extra information to help me here?

Edit:
Two "new" errors occured:
![image](https://user-images.githubusercontent.com/83671398/230400803-a2686931-2a63-4b53-8ef6-b7649da0fbb4.png)

**Comment by marcelklehr** [comment, marked as answer]
I cannot help you with this. It seems unrelated to recognize.
